.pre-commit-config.yaml

```diff
@@ -30,9 +30,18 @@ repos:
     rev: "v0.761"
     hooks:
       - id: mypy
-        exclude: ^semgrep/tests/.+$
+        exclude: ^semgrep\/tests\/.+$
```

Comment:
whaaat how did this work before? 🤯

Reply:
my guess is that the pre-commit regex dialect is ok with forward slashes being unescaped?